src/sentry/issues/priority.py

```diff
@@ -70,6 +77,34 @@ def update_priority(
     if group.type == MetricIssue.type_id:
         update_incident_activity_based_on_group_activity(group, priority)
 
+    # create a row in the GroupOpenPeriodActivity table
+    open_period = get_latest_open_period(group)
+    if open_period is None:
+        logger.error("No open period found for group", extra={"group_id": group.id})
```

Comment:
thoughts on also sending a metric here so we can add it to a dashboard? (then use this log to figure out which open periods don't exist).